orm/tests/columnTypesTests/int/int.test.ts

```diff
@@ -89,62 +97,670 @@ AllIntsSuite.before(async (context) => {
   const allIntsTable = context.allIntsTable!;
 
   await allIntsTable.insert(allPositiveFields).execute();
-
   const insertedValues = await allIntsTable.insert(mixedFields).all();
 
-  // assert insertedValues has all expected values
   const fullSelectResponse = await allIntsTable.select().all();
-  // assert table has all expected values inserted
-  const partialSelectResponse = await allIntsTable.select({
-    notNullInt: allIntsTable.notNullInt,
-    intWithDefault: allIntsTable.intWithDefault,
-  }).all();
+  const partialSelectResponse = await allIntsTable
+    .select({
+      notNullInt: allIntsTable.notNullInt,
+      intWithDefault: allIntsTable.intWithDefault,
+    })
+    .all();
+
+  assert.is(insertedValues.length, 1);
+  assert.equal(insertedValues[0], mixedFields);
 
   assert.is(fullSelectResponse.length, 2);
+  assert.equal(
+    fullSelectResponse.filter((it) => it.serialInt === allPositiveFields.serialInt)[0],
+    allPositiveFields,
+  );
+  assert.equal(
+    fullSelectResponse.filter((it) => it.serialInt === mixedFields.serialInt)[0],
+    mixedFields,
+  );
+
   assert.is(partialSelectResponse.length, 2);
-  assert.is(partialSelectResponse.filter((it) => it.notNullInt === 0)[0].intWithDefault, -1);
+  assert.is(
+    partialSelectResponse.filter((it) => it.notNullInt === allPositiveFields.notNullInt)[0]
+      .intWithDefault,
+    allPositiveFields.intWithDefault,
+  );
+  assert.is(
+    partialSelectResponse.filter((it) => it.notNullInt === mixedFields.notNullInt)[0]
+      .intWithDefault,
+    mixedFields.intWithDefault,
+  );
+});
 
-  // update by 1 strategy
-  // logic
+AllIntsSuite('Insert all required fields to table', async (context) => {
+  const allIntsTable = context.allIntsTable!;
 
-  // same select
-  // assert insertedValues has all expected values
-  //   const fullSelectResponse = await allIntsTable.select().all();
-  //   // assert table has all expected values inserted
-  //   const partialSelectResponse = await allIntsTable.select({
-  //     notNullInt: allIntsTable.notNullInt,
-  //     intWithDefault: allIntsTable.intWithDefault,
-  //   }).all();
+  await allIntsTable.insert(requiredPositiveFields).execute();
+  const insertedValues = await allIntsTable.insert(requiredMixedFields).all();
 
-  //   assert.is(fullSelectResponse.length, 2);
-  //   assert.is(partialSelectResponse.length, 2);
-  //   assert.is(partialSelectResponse.filter((it) => it.notNullInt === 0)[0].intWithDefault, -1);
+  const fullSelectResponse = await allIntsTable.select().all();
+  const partialSelectResponse = await allIntsTable
+    .select({
+      notNullInt: allIntsTable.notNullInt,
+      notNullIntWithDefault: allIntsTable.notNullIntWithDefault,
+      notNullUniqueInt: allIntsTable.notNullUniqueInt,
+    })
+    .all();
 
-  // delete by 1 strategy
+  assert.is(insertedValues.length, 1);
+  assert.equal(insertedValues[0], {
+    ...requiredMixedFields,
+    serialInt: 2,
```

Comment:
if you will change `requiredMixedFields` model values, then this test will be broken. I guess you need to get value from `requiredMixedFields`

ex. ```serialInt: requiredMixedFields.serialInt```

Same for all test in this file

Reply:
This is because the serialInt field is not a required field, so it's not in requiredMixedFields. After inserting requiredMixedFields I check that serialInt field was added by default.

Do I need to add serialInt to requiredMixedFields and remake it according to the example ?